src/deploy/functions/deploy.ts

```diff
@@ -0,0 +1,53 @@
+import * as gcp from "../../gcp";
+import * as clc from "cli-color";
+import { setGracefulCleanup } from "tmp";
+import { logBullet, logSuccess, logWarning } from "../../utils";
+import * as prepareFunctionsUpload from "../../prepareFunctionsUpload";
+
```

Comment:
fwiw, I've liked trying to group these in "node and npm modules" then "local modules" groups

Reply:
Done.